src/composables/multisigAccounts.ts

```diff
@@ -61,10 +61,11 @@ const activeMultisigAccountId = ref<Encoded.AccountAddress>();
 const activeMultisigNetworkId = ref('');
 const isAdditionalInfoNeeded = ref(false);
 
+const { aeActiveNetworkPredefinedSettings } = useAeNetworkSettings();
```

Comment:
The child composables should be initiated inside of the parent composable - not outside it's scope.

Reply:
When being initiated inside of the parent composable it is not being updated correctly